### DISCUSSION TITLE
How to run on another port?

### DISCUSSION THREAD

**Discussion by aalmans** [ItzCrazyKns/Vane#139] [Q&A] [answered]
It is conflicting with Open WebUI on port 3000. Can Perplexica run on another port?

**Comment by ItzCrazyKns** [comment, marked as answer]
Just change the ports in the compose file and re-build the images.

**Reply by tristam15** [reply]
I changed them in the docker compose file, when I run the localhost:3001, it says "CANNOT GET /" error. It is not working. 

Any help? 
<img width="103" alt="image" src="https://github.com/user-attachments/assets/dea232ac-0ece-4948-877c-c28670ceb8fd">

**Reply by ItzCrazyKns** [reply]
But that is for the backend, what port did you use for the frontend?
